app/workspace/page.tsx

```
@@ -262,6 +262,163 @@ function WorkspaceContent() {
+              </svg>
+            </button>
+          </div>
+
+          {/* Dropdown Menu */}
+          {!sidebarCollapsed && (
+            <div className="absolute top-full left-0 mt-2 ml-4 bg-white rounded-lg shadow-lg border border-gray-200 py-2 min-w-[200px] fade-in">
+              <button
+                onClick={handleCreateDocument}
+                className="w-full text-left px-4 py-2 hover:bg-gray-100 transition-colors"
+              >
+                ➕ New Document
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)